_posts/2020-01-13-release-r118/index.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title-short: "Snowplow R118 with new bad row format"
+title: "Snowplow R118 beta release with new bad row format"
+tags: [snowplow, enrichment, bad-rows, release]
+author: Benjamin Benoist
+category: Releases
+permalink: /blog/2020/01/13/snowplow-release-r118-badrows/
+---
+
+We are excited to release [Snowplow R118 Morgantina][snowplow-release], named after the
+[archaeological site in east central Sicily, southern Italy](https://en.wikipedia.org/wiki/Morgantina).
+
+This Snowplow beta release includes the long-awaited new bad row format. We have updated the microservices that make up the Snowplow pipeline to emit much more highly structured bad rows, that should be significantly easier to query and debug: making it simpler for companies running Snowplow to realibly monitor for data quality issues, spot when they arise, and then quickly diagnose the source of type of problem tha that has emerged, the event or entity that it impacts, the size of the problem and the app it has occurred on. For more information on the thinking behind this new functionality, see the [original RFC][RFC].
+
+In addition, this release also includes important refactoring in the libraries that we use (e.g. `cats` instead of `scalaz` and `circe` instead of`json4s`),
+a bump of [Beam](https://beam.apache.org/) version to 2.11.0,
+an [updated](https://snowplowanalytics.com/blog/2019/08/09/iglu-scala-client-0.6.0-released/) [Iglu Client](https://github.com/snowplow/iglu-scala-client)
+with different validator and fixed caching mechanism,
+as well as an improvement to the referer parser enrichment.
+
+[Scala Stream Collector](https://github.com/snowplow/snowplow/tree/master/2-collectors/scala-stream-collector)
+and the enrich jobs versions ([Stream Enrich](https://github.com/snowplow/snowplow/tree/master/3-enrich/stream-enrich) and
+[Beam Enrich](https://github.com/snowplow/snowplow/tree/master/3-enrich/beam-enrich)) are bumped to `1.0.0`.
+
+As introduced in the [RFC](https://discourse.snowplowanalytics.com/t/rfc-making-the-snowplow-pipeline-real-time-end-to-end-and-deprecating-support-for-batch-processing-modules/3018),
+we stop supporting Spark from this release, in favor of streaming.
```

Comment:
```suggestion
we are no longer adding new features to Spark from this release. This release, and all subsequent, will focus on our streaming solution.
```
Just added a bit more emphasis to reduce the chances of this being missed. We were at one point going to add the capability to Spark Enrich. Did we back out of this?